app/src/main/java/net/frju/flym/ui/main/FeedSearchDialog.kt

```diff
@@ -144,6 +148,12 @@ class FeedSearchDialog(context: Context, private val defaultFeeds: List<SearchFe
                 doAfterFiltering()
             }
         }
+
+        if (search.isNotBlank()) {
+            this.setSearchHint(search)
```

Comment:
instead of doing it with the search hint, I believe you can do "searchBox.text = search" in onCreate()

Reply:
I already tried using this.searchBox.setText() in FeedSearchDialog.kt, I get a NullPointerException for searchBox.
I just tried doing the same in MainActivity.kt  with :
```
if (intent?.action.equals(Intent.ACTION_VIEW)) {
    var search : String = intent?.data.toString()
    var searchDialog = FeedSearchDialog(this, search)
    searchDialog.searchBox.setText(search.subSequence(0, search.length) 
    searchDialog.show()
}
```
and same result, NullPointerException for searchBox

You can't use `searchBox.text = search` because searchBox.text is a EditText.

Reply:
I guess you got an NPE because you didn't access to the field in the good method (not inflated yet). I actually already access this field in the onCreate of the FeedSearchDialog and don't have NPE

Reply:
Yes the searchBox is initialized during searchDialog.show() so changing the text just after showing it works perfectly